guests-selector/utils/constants.js

```diff
@@ -0,0 +1,9 @@
+export const CLASS_NAMES = {
```

Comment:
@samuelsilvapt we should be setting a prefix and concat to make this naming less error prone.

```
const baseName = 'irnmn-guests-selector`

...
roomContainer: `${baseName}__room-container`